### ISSUE TITLE
[Feature Request]: Allow miniplayer when listening to "content made for kids"

### ISSUE THREAD

**Issue by eitan-j** [th-ch/youtube-music#2557]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

When listening to certain content considered "made for kids",  if I attempt to go to any other page such as them main menu, the song will pause and show the message "Miniplayer is off for content made for kids. Tap play to resume" and a link labeled "Learn More" which links to https://support.google.com/youtube/answer/9632097. When clicking the play button to resume the song, it enlarges the song back to the main player screen.

### Proposed Solution

Attempting to go to other pages while viewing "made for kids" content should result in the song continuing to play in the miniplayer like any other song.

### Alternatives Considered

The help page indicates that this behavior is to comply with the COPPA law. Since COPPA is a US law, I tried the same thing while connected via a VPN through Sweden. The problem still occurred.

### Additional Information

This was also asked by @Lockszmith-GH in https://github.com/th-ch/youtube-music/discussions/2341

Example track: https://music.youtube.com/watch?v=_rHjCtNARH0

**Comment by SirDufus** [comment]
Canadian checking in. Not sure why US laws are imposed on users outside the country. Also, there seems to be no reasoning behind the error. This morning, the music stopped playing while listening to the Supermix because, apparently, Carole King is content made for kids now.

Google did not imnplement this correctly and it's affecting international users as welll as non kid content. A work around plugin would be fantastic.

My 02 cents

**Comment by robhogg** [comment]
Yes, beyond the appropriateness of the feature itself as a response to COPPA, which "prohibits unfair or deceptive acts or practices in connection with the collection, use, and/or disclosure of personal information from and about children on the Internet" (seemingly having nothing to do with display of videos in a miniplayer) or the enforcement of COPPA outside the USA, the process for identification of content as "made for kids" is clearly broken.

Yesterday, it flagged Weber's **Invitation to the Dance** as made for kids (It's not exactly The Young Person's Guide to the Orchestra), and today Carole King's **It's too Late** (with lyrics about a break-up). Strangely, **The Muppet's Mayhem** album by Dr Teeth and the Electric Mayhem isn't flagged. Nor is Roy Bailey's album **Why Does It Have to Be Me?**, which he wrote for children. ¯\_(ツ)_/¯

**Comment by MulverineX** [comment]
It'd be nice if this was still implemented as a plugin, even though it is YouTube's bs

**Comment by Bluscream** [comment]
Yep, same issue here!

![](https://files.catbox.moe/8fg0u2.png)

**Comment by duncte123** [comment]
Yeah this feature makes no sense for a music app. Especially when the "learn more" link leads to an article talking about videos.
I half expected the "bypass age restriction" plugin to tackle this problem

(Not a US citizen myself nor do I live close to the country, why does this even apply to me?)

**Comment by colineberly** [comment]
Parroting in for this as well. Is there some sorf of script to bypass this nonsense? Playing a "kids" song it basically disables you from navigating around and adding more songs. Funniest part is some of these songs can be either or. Whoever added this feature is a moron.

**Comment by Sosenteam** [comment]
This problem is also bothering me... It's also happening occasionally for non-"kids" tracks (White Stripes, etc.)

**Comment by PauloDand** [comment]
An addon to block this limitation would be the best option.
I'm having issues with songs that are not even close to be kids oriented.
